developer-docs/latest/concepts/models.md

```diff
@@ -168,14 +168,17 @@ The options key on the model-json states.
 
 - `populateCreatorFields`: Configure whether the API response should include `created_by` and `updated_by` fields or not. Accepts a `boolean`. The default value is `false`.
 
-**Path —** `User.settings.json`.
+- `draftAndPublish`: Enable the draft feature. Accepts a `boolean`. The default value is `false`.
```

Comment:
No sure if 
- "draft feature"
- or "draft and publish feature".
- maybe "_draft and publish_ feature"

Reply:
> Enable the draft and publish feature. Accepts....